doc/release-notes-1.0.0.rst

```diff
@@ -0,0 +1,302 @@
+
+BSD library
+-----------
+
+* Updated bsdlib to version 0.3.1.
+* Introduced a new header bsdlib.h to be used by the application to initialize and shutdown the library.
+* Library initialization during system initialization (``SYS_INIT``) is now optional, and controlled via ``Kconfig``. The default behavior is unchanged.
+
+Secure Partition Manager (SPM) library
+--------------------------------------
+
+* Added random number secure service, providing access to the RNG hardware from the non-secure firmware.
+* Non-Secure callable support:
+
+  * A secure_services module is now available over secure entry functions. This means:
+
+    * :file:`secure_services.c` resides in secure firmware (SPM).
+    * :file`secure_services.h` declares functions that can be called from non-secure firmware.
+
+  * SPM now exposes secure entry functions by default.
+  * Added reboot as a secure service. The reboot secure service is called when the non-secure firmware calls sys_reboot().
+
+* PWM0-3 added as non-secure.
+
+
+Common libraries
+================
+
+* Added the following libraries:
+
+  * :ref:`ppi_trace` - enables tracing of hardware peripheral events on pins.
+
+Enhanced Shockburst
+-------------------
+
+* Added support for nRF52811.
+
+Download Client
+---------------
+
+* Added IPv6 support, with fallback to IPv4.
+* Added HTTPS support. The application must provision the TLS security credentials.
+* Several improvements to buffer handling and network code.
+* Library now runs in a separate thread.
+
+
+Crypto
+======
+
+*
+*
+
+nRF BLE Controller
+==================
+
+* Added support for the nRF BLE controller 0.2.0-prealpha. Includes drivers to access HCI, flash, clock control, and entropy hardware.
+
+Subsystems
+==========
+
+Bluetooth Low Energy
+--------------------
+
+* Added the following samples:
+
+  * :ref:`central_bas` - demonstrates how do use the :ref:`bas_c_readme` to receive battery level information from a compatible device.
+  * :ref:`shell_bt_nus` - demonstrates how to use the :ref:`shell_bt_nus_readme` to receive shell commands from a remote device.
+
+* Added the following libraries:
+
+  * :ref:`bas_c_readme` - used to retrieve information about the battery level from a device.
+  * :ref:`shell_bt_nus_readme` - allows for sending shell commands from a host to the application.
+
+* Added :ref:`ble_console_readme` - a desktop application that can be used to communicate with an nRF device over *Bluetooth* Low Energy using the :ref:`shell_bt_nus_readme`.
+
+NFC
+---
+
+*
+
+Partition Manager
+-------------
+
+* Partition Manager can now implicitly or explicitly assign a HEX file to a partition.
+* :ref:`ug_pm_static` of upgradable images is now supported.
+
+
+nRF Desktop
+===========
+
+* The nrf_desktop reference implementation is moved from the ``samples/`` folder to ``applications/``.
+* The nrf_desktop configuration channel now allows data to be exchanged between the device and host in both directions.
```

Comment:
@jhn-nordic should we add details of what was changed before project is released?